src/functionalTest/java/uk/gov/hmcts/dts/fact/admin/AdminCourtContactEndpointTest.java

```diff
@@ -156,14 +156,35 @@ public void shouldBeForbiddenForRetrievingContactTypes() {
         assertThat(response.statusCode()).isEqualTo(FORBIDDEN.value());
     }
 
-    private void verifyCourtContactsAfterUpdate() {
+    @SuppressWarnings("PMD.DataflowAnomalyAnalysis")
+    private void verifyCourtContactsAfterUpdate(final List<Contact> expectedContacts) {
+        // Call the standard /courts/{slug} endpoint
         var response = doGetRequest(COURTS_ENDPOINT + BIRMINGHAM_CIVIL_AND_FAMILY_JUSTICE_CENTRE_SLUG);
         assertThat(response.statusCode()).isEqualTo(OK.value());
+        final Court court = response.as(Court.class);
+
+        // Call the contact types endpoint so we can map the type name from the standard courts endpoint to the type ID
+        // from the expected results
+        response = doGetRequest(CONTACT_TYPES_FULL_PATH, Map.of(AUTHORIZATION, BEARER + authenticatedToken));
+        assertThat(response.statusCode()).isEqualTo(OK.value());
+        final List<ContactType> contactTypes = response.body().jsonPath().getList(".", ContactType.class);
+        final Map<Integer, ContactType> contactTypeMap = contactTypes.stream()
+            .collect(toMap(ContactType::getId, type -> type));
 
-        Court court = response.as(Court.class);
         List<uk.gov.hmcts.dts.fact.model.Contact> contacts = court.getContacts();
-        assertThat(contacts).hasSizeGreaterThan(1);
-        assertThat(contacts.stream()).anyMatch(contact -> contact.getNumber().equals(TEST_NUMBER));
+        assertThat(contacts).hasSize(expectedContacts.size());
+
+        // Check the results coming back from the standard courts endpoint (i.e. from the front end) contain the expected
+        // information in the expected order
+        for (int i = 0; i < expectedContacts.size(); i++) {
+            final Integer expectedContactTypeId = expectedContacts.get(i).getTypeId();
+            final String expectedContactType;
```

Comment:
Can prob move this declaration of the string into line 183? It doesn't look like it's used anywhere else. Wasn't too sure why it wasn't initialised with a value also lol